unity/CAVE/Assets/W3D_Translator/Classes.cs

```diff
@@ -1516,4 +1805,37 @@ public class Disc
         [XmlAttribute(AttributeName="radius-inner")]
         public double innerRadius;
     }
-}
\ No newline at end of file
+
+
+    /********** BASE CLASS              ***********/
+
+
+    [Serializable]
+    public class W3D
+    {
+        public string pprint()
+        {
+            return JsonUtility.ToJson(this, true);
+        }
+
+        static public Color ConvertColor(string colorString)
+        {
+            string[] strings=colorString.Trim(new Char[] { ' ', '(', ')' }).Split(",");
+            return new Color(
+                float.Parse(strings[0]) / 255,
+                float.Parse(strings[1]) / 255,
+                float.Parse(strings[2]) / 255
+            );
+        }
+
+        static public Vector3 ConvertVector3(string vectorString)
+        {
+            string[] strings=vectorString.Trim(new Char[] { ' ', '(', ')' }).Split(",");
+            return new Vector3(
+                float.Parse(strings[0]),
+                float.Parse(strings[1]),
+                float.Parse(strings[2])
+            );
+        }
+    }
```

Comment:
I get the pretty print, that's a nice thing to have on all child classes. These static methods seem unrelated to their underlying data. Does a camera really need `ConvertColor`? Does a color really need `ConvertVector3`? Can you make these util functions within the same namespace or something? Or do they need to be wrapped in a class structure?

Reply:
Everything needs to be inside of a class in c#... I could add a methods class? Or just add them inside the `CLI` class in `CLI.cs`? That file is going to get pretty messy though so I'm not sure what's best

Reply:
Eh, I'd just keep them in here. I was mostly curious